dotcom-rendering/src/web/components/Callout/FormField.tsx

```diff
@@ -232,7 +231,6 @@ export const FormField = ({
 				</div>
 			);
 		default:
-			logger.error(`Invalid field ${type} provided for callout`);
```

Comment:
@sophie-macmillan I also had to remove the logger as this is not supported in DCR